pages/chat/[[...roomId]].js

```diff
@@ -136,7 +136,7 @@ export default function RoomId() {
         };
     });
 
-    const invites = _.sortBy([...matrix.invites.values()], sortRooms);
+    const invites = _.sortBy([...matrix.invites.values()], sortRooms).filter(invite => !invite.meta);
```

Comment:
Can you add a comment what this `.filter(invite => !invite.meta);` call does "in layman terms"?
Is it there to show all invites, _except_ for the ones that we assume to be part of one application?

Which makes me wonder: Wouldn't normal "chats" that we start from the /spaces app also have a `dev.medienhaus.meta` event in the future? Is this filter accurate then?

Reply:
I've added a comment.

> Which makes me wonder: Wouldn't normal "chats" that we start from the /spaces app also have a dev.medienhaus.meta event in the future? Is this filter accurate then?

Yeah this is sort of an ongoing discussion, whether we need a `dev.medienhaus.meta` event for chat rooms or not. I personally don't see the advantage of "spamming" chat rooms with unnecessary events, or to be able to verify the origin? (as in coming from the /spaces app) but I don't really mind either way.

Reply:
> Which makes me wonder: Wouldn't normal "chats" that we start from the /spaces app also have a dev.medienhaus.meta event in the future? Is this filter accurate then?

No, we should not add any `dev.medienhaus.meta` events to **chat** rooms.

Chat rooms are considerably well defined, and _none of our business_; we _use_ chat rooms, but we don’t want to re-invent chat rooms or break any compatibility with, well, *any* matrix chat room from any matrix chat client, like for example [element-web](https://github.com/vector-im/element-web).

Reply:
Cool, agreed then. 👍  Let's just add a short comment on what the filtering is supposed to do. Feel free to leave a link to _this_ very discussion here on GitHub inside the code.